src/features/home/components/modules/HomeModule.web.test.tsx

```diff
@@ -188,7 +188,7 @@ describe('<HomeModule />', () => {
 
     await act(async () => {})
 
-    expect(screen.getByText('La nuit des temps')).toBeInTheDocument()
+    expect(screen.getByText('I want something more')).toBeInTheDocument()
```

Comment:
`La nuit des temps` is not visible in the `FlashList` snapshot since the `FlashList` behavior is to render only the elements on the screen